### ISSUE TITLE
cosmic-ray init is overly "helpful" by creating a new config if none exists

### ISSUE THREAD

**Issue by rob-smallshire** [sixty-north/cosmic-ray#326]
I ran `cosmic-ray exec` when I was unknowingly in the wrong directory. I was very surprised when Cosmic Ray presented me with the menu for creating a new config. So much so, that I had to read the help to check I was actually running the right command, and then when I established that I was, read the source to figure out why `exec` was invoking `new_config`.

The culprit is this line:

```
    if not os.path.exists(config_file):
        handle_new_config(args)
```

This is just confusingly too helpful. I'd much prefer that Cosmic Ray terminated with an error telling me that the specified config file does not exist. That way, I would have quickly figured out that I was in the wrong directory.   Ideally, Cosmic Ray would exit with EX_NOINPUT.

Aside: There's also a race condition between the file existence check and the reading of the config file.  It would be more robust to use EAFP than LBYL here, and have `load_config()` raise appropriate exceptions.

**Comment by abingham** [comment]
After using this for a while, I tend to agree. We added this feature during the fever of a sprint, largely because it was easy. I'll be happy to remove it.

> There's also a race condition

There will be a race condition no matter how we slice it. If `load_config` raised an exception and we started the menu process in response, a config could potentially be created in the interim. But since we're getting rid of it, I guess it's not much of a concern :)
